openQA.spec

```
@@ -238,6 +239,16 @@ Group:          Development/Tools/Other
 Documentation material covering installation, configuration, basic test writing, etc.
 Covering both openQA and also os-autoinst test engine.
 
+%package auto-upgrade
+Summary:        Automatically upgrade and reboot an openQA system where required
+Group:          Development/Tools/Other
+Requires:       curl
+Requires:       rebootmgr
+
+%description auto-upgrade
+Use this package to install and enable a SystemD Service upgrading and rebooting
+an openQA system where required.
```

```suggestion
Use this package to install and enable a systemd service for nightly upgrading and rebooting
the system if devel:openQA packages are stable.
```

(systemd is *always* lower case.)